src/core/thread/mle.cpp

```diff
@@ -1733,6 +1739,10 @@ ThreadError Mle::HandleDataResponse(const Message &aMessage, const Ip6::MessageI
             mNetif.GetActiveDataset().Set(activeTimestamp, aMessage, offset + sizeof(tlv), tlv.GetLength());
         }
     }
+    else
+    {
+        VerifyOrExit(update, ;);
```

Comment:
This is the case where no active dataset exists in the received message.  In this case, why do we only process the pending dataset on first attach?